pennylane_orquestra/utils.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains utility functions for the PennyLane-Orquestra plugin.
+
+The original implementation of the ``_process_wires`` and
+``_terms_to_qubit_operator`` functions is a part of the PennyLane-QChem
+library.
```

Comment:
maybe good to add a link to the GitHub location of the function?

Reply:
Think that could be nice, though could the location break if the function is moved?

Reply:
yep 😆